cli/test/smokehouse/test-definitions/fps-scaled.js

```diff
@@ -46,8 +46,8 @@ const expectations = {
         ],
       },
       screenshot: {
-        height: 2000,
-        width: 824,
```

Comment:
The bounding boxes for elements do not line up with these image dimensions. I think the previous expectations for this smoke test were correct:

![Screenshot 2023-04-18 at 3 01 14 PM](https://user-images.githubusercontent.com/6752989/232914826-d64d8310-3a19-4ec5-b280-447dbdac1267.png)

Reply:
@alexnj I'm still seeing this issue and I think the old expectations were correct. Try adding an a11y failure to the `<h1>` element (e.g. make it `<h1 role="adsf'>`) and then you will see this in the report.